doc/processes/release_process.md

```diff
@@ -116,13 +116,19 @@ determination can be made to either:
 	* Apply a targetted fix to the release branch and retag, or
 	* Rebase the release branch on the master branch if the 
 	changes that have gone in between the initial RC tag and now are safe.
-1. Retag the `RCx` level as `openj9-#releaseNumber#`.  For the `0.8` release this 
-will be `openj9-0.8`.
+1. Retag the `-rcX` level as `openj9-#releaseNumber#`.  For the `0.8.0` release this 
+will be `openj9-0.8.0`.
 1. Create the [github release](https://help.github.com/articles/creating-releases/)
 corresponding to the tagged level.  The release should link to the Eclipse Release 
 document, the release issue, and the AdoptOpenJDK download links.
 1. Open an Eclipse Bugzilla requesting the branch be marked `protected` to prevent 
 commits after the release is complete.
 
-The `java -version` should now show the tagged level.
+For a milestone release, the tag will be of the form: `openj9-0.8.0-m1`.
+
+Note, use annotated tags (`git tag -a <tag name>`) as they are treated specially by
+git compared to simple tags.
+
+The `java -version` should now show the tagged level due to the `${java.vm.version}`
+property being set to the tag  and the javacore will also show the tag.
```

Comment:
This doesn't read well. I'd make "and the javacore will also show the tag" into another sentence, or at least put a comma before.